packages/server/src/admin/invite.ts

```diff
@@ -54,7 +54,7 @@ export async function inviteHandler(req: Request, res: Response): Promise<void>
   } catch (err: any) {
     logger.info('Error inviting user to project', {
       project: projectId,
-      error: err,
+      error: err.toString(),
```

Comment:
You may consider using `normalizeErrorString`: https://github.com/medplum/medplum/blob/main/packages/core/src/outcomes.ts#L276